config/jobs/kubernetes-csi/external-resizer/external-resizer-config.yaml

```diff
@@ -0,0 +1,103 @@
+# generated by gen-jobs.sh, do not edit manually
+
+presubmits:
+  kubernetes-csi/external-resizer:
+  - name: pull-kubernetes-csi-external-resizer
+    always_run: true
+    optional: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    annotations:
+      test-grid-dsashboards: sig-storage-csi
+      testgrid-tab-name: pull-kubernetes-csi-external-resizer
+      description: Kubernetes-CSI pull non-alpha job in external-resizer
+    spec:
+      containers:
+      # We need this image because it has Docker in Docker and go.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190819-0b0980f-master
+        command:
+        - runner.sh
+        args:
+        - ./.prow.sh
+        env:
+        - name: CSI_PROW_KUBERNETES_VERSION
+          value: "1.15.0"
+        - name: CSI_PROW_TESTS
+          value: "sanity serial parallel"
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            cpu: 2000m
+  - name: pull-kubernetes-csi-external-resizer-unit
+    always_run: true
+    optional: false
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
+      preset-service-account: "true"
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+    annotations:
+      test-grid-dsashboards: sig-storage-csi
```

Comment:
```suggestion
      test-grid-dashboards: sig-storage-csi
```

Reply:
Good catch, I've updated the script and regenerated all the configs